core/controller.go

```diff
@@ -216,13 +230,34 @@ func (c *Controller) ListStreamHandler(w http.ResponseWriter, r *http.Request, _
 		return
 	}
 	dto := []*SummariseDTO{}
+
+	// active streams
 	for key, stream := range c.streams {
+		aliasname := ""
+		for name, id := range c.alias {
+			if id == stream.ID {
+				aliasname = name
+				key = name
```

Comment:
Why are we setting the key to name?

Reply:
Preference is to display the alias over the uuid for the suggested path.

Reply:
https://play.golang.org/p/wjua_Le43ql

I don't think it achieves what you would like to achieve here

Reply:
It did achieve what I wanted, but it was not obvious. I should have used a "newkey" variable instead. Its purpose is to decide if it can use the alias in the URL response. I'll make it more obvious.